imports/plugins/core/hydra-oauth/server/init.js

```diff
@@ -7,6 +7,6 @@ Hooks.Events.add("afterCoreInit", () => {
   Reaction.addRolesToGroups({
     allShops: true,
     groups: ["guest", "customer"],
-    roles: ["account/login"]
+    roles: ["account/login", "not-found"]
```

Comment:
Since the IDP defines the Not Found registry route, it needs to grant the access to it.